src/builder.cr

```diff
@@ -34,6 +38,17 @@ module Mint
         end
       end
 
+      unless artifacts.assets.empty?
+        FileUtils.mkdir Path[DIST_DIR, ASSET_DIR].to_s
+
+        terminal.puts "#{COG} Writing assets..."
+
+        artifacts.assets.each do |asset|
+          puts "  #{ARROW} #{asset.filename}"
+          File.write Path[DIST_DIR, ASSET_DIR, asset.filename.to_s], asset.real_path
```

Comment:
Shouldn't you write the original file contents, instead of its path?

```suggestion
          File.open(asset.real_path) do |io|
            File.write Path[DIST_DIR, ASSET_DIR, asset.filename.to_s], io
          end
```

Reply:
Or, if we don't care about copying some borked permissions, this could be simplified to:

```suggestion
          File.copy asset.real_path, Path[DIST_DIR, ASSET_DIR, asset.filename.to_s]
```

Reply:
Yeah, I totally missed that :man_facepalming: Thanks! :+1: :1st_place_medal:

Reply:
ftw, the specs are not properly testing that atm.